test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex route
```

Comment:
I spend couple of days to implement these steps but had some difficulties in implementation though it looks more generic. 

> And "ruby-ex" exposes route "foo"

Here i am seeing extra layer of validation. Implementation would be look like this 
```
func (m *Minishift) exposeRoute(serviceName, nameSpace) error {
	cmdOut, cmdErr, cmdExit := m.runner.RunCommand("project")
	if len(cmdErr) != 0 || cmdExit != 0 || !strings.Contains(cmdOut, nameSpace){
		return fmt.Errorf("Command did not execute successfully or in the wrong namespace. cmdExit : %d, cmdErr: %s, cmdOut : %s", cmdExit, cmdErr, cmdOut) //Extra layre of validation
	} else {
		cmdOut, cmdErr, _ := m.runner.RunCommand("expose svc/" + serviceName)
		if len(cmdErr) != 0 {
			return fmt.Errorf("Command did not execute successfully. cmdErr: %s", cmdErr)
		}
	}
	return nil
}
```
To avoid this we can use When executing "oc expose svc/ruby-ex" succeeds as it was previously.
> Then status code of HTTP request to "/" of "foo" is equal to "200"
And body of HTTP request to "/" of "foo" contains "Welcome to your Ruby application on OpenShift"

I tried some of ways/tricks to implement but it hard to manage as part of argument is concerned though the scenario steps looks more generic. I used some of other commands such as ```-o=custom-columns=URL:.spec.host``` and ```-o=custom-columns=NAME:.metadata.name,HOST:.spec.host``` but did get the proper match of arguments to implement the step. 

i modified the feature file keeping rest changes intact. Scenario would look like this 
```
Scenario: User can create route for ruby-ex to make it visiable outside of the cluster
   Given Minishift has state "Running"
    When executing "oc expose svc/ruby-ex" succeeds
    Then stdout should contain
     """
     exposed
     """
    When status code of HTTP request to service "ruby-ex" of namespace "ruby" is equal to "200"
    Then body of HTTP request to service "ruby-ex" of namespace "ruby" contains "Welcome to your Ruby application on OpenShift"
```